lib/engine-jit/src/artifact.rs

```diff
@@ -141,20 +137,26 @@ impl JITArtifact {
     }
 
     /// Deserialize a JITArtifact
-    pub fn deserialize(jit: &JITEngine, bytes: &[u8]) -> Result<Self, DeserializeError> {
+    pub unsafe fn deserialize(jit: &JITEngine, bytes: &[u8]) -> Result<Self, DeserializeError> {
```

Comment:
`unsafe` functions should have a `# Safety` section saying how the function should be used (i.e. why is it unsafe, what does the user need to manually check)

Reply:
Fixed